### PR TITLE
Size the "Give a review" component based on the screen

### PR DESCRIPTION
**Pull request by dmarcoux**
## Before

| Desktop  |  Mobile |
|---|---|
| before-desktop  | before-mobile  |

## Now

| Desktop  |  Mobile |
|---|---|
| now-desktop  | now-mobile  |

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Review Modal Size for "Give a review" (S)](https://trello.com/c/vUq1tNYT/2125-review-modal-size-for-give-a-review-s)

File: src/api/app/assets/stylesheets/request_show_redesign/add_review.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #review-form-collapse {
   padding: inherit;
-  left: 0;
+  right: 1.5rem;
```
Comment (dmarcoux) [inline src/api/app/assets/stylesheets/request_show_redesign/add_review.scss:3]: This is aligning this element with the button `Review`.

**Review by danidoni — APPROVED** [review]

File: src/api/app/assets/stylesheets/request_show_redesign/add_review.scss (hunk 1 above)
Comment (eduardoj) [reply src/api/app/assets/stylesheets/request_show_redesign/add_review.scss:3]: I would go with `right: 0;`. In your examples, you are using a width of 412px. When migrating to Bootstrap, we introduced the mobile-first approach using a width of 320px. Another thing is that it makes sense nowadays.

Comment (dmarcoux) [reply src/api/app/assets/stylesheets/request_show_redesign/add_review.scss:3]: Okay, let's go with `right: 0;`. I find both options good.

**Review by eduardoj — APPROVED** [review]

**Pull request merged**